presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java

```diff
@@ -54,6 +54,41 @@ public long getNumberOfValues()
         return numberOfValues == null ? 0 : numberOfValues;
     }
 
+    public boolean hasAverageValueSizeInBytes()
+    {
+        // TODO: determine the size of decimals
+        return hasNumberOfValues() &&
+                numberOfValues > 0 &&
+                (integerStatistics != null ||
+                        booleanStatistics != null ||
+                        doubleStatistics != null ||
+                        stringStatistics != null ||
+                        dateStatistics != null);
+    }
+
+    public long getAverageValueSizeInBytes()
+    {
+        if (!hasAverageValueSizeInBytes()) {
```

Comment:
The caller (OrcRecordReader) will fall back to the original heuristics; i.e., 1024 rows at most per load and adjusts the number of rows to load according to the previous loaded blocks. Do we wanna make this an assert so that caller should always call `hasAverageValueSizeInBytes` first?